vllm/lora/models.py

```
@@ -215,9 +216,22 @@ def from_local_checkpoint(
         new_embeddings_bin_file_path = os.path.join(lora_dir,
                                                     "new_embeddings.bin")
 
+        tensorizer_config = kwargs.get("tensorizer_config")
```

Why is this using `kwargs` instead of an argument with a default value? Adding `kwargs` to a function is significant since it will start silently accepting and ignoring incorrect parameter names passed to the function if you don't also go out of your way to check for extraneous entries in it. Is there a reason you need to use `**kwargs` over `tensorizer_config=None`?

---

The motivation was to along the lines of "I want to include my tensorizer hook, but I don't want to be as intrusive as to ask the developers to include my `tensorizer_config` as directly as an acknowledged parameter in their LoRA API with a docstring and everything. I'll instead only be intrusive enough as to ask them to allow `**kwargs`, setting a precedent for others to hook in their own stuff unintrusively.".

This has made me see though that this is the opposite of unintrusive; it's harming the API itself to allow `**kwargs` like this and making the LoRA machinery harder to maintain and setting a precedent of LoRA logic having to handle any number of random, undocumented parameters. I'll explicitly introduce `tensorizer_config` as a parameter.